webrtc-stats.html

```diff
@@ -1982,7 +1982,7 @@
           <pre class="idl">dictionary RTCDataChannelStats : RTCStats {
              DOMString           label;
              DOMString           protocol;
-             long                datachannelid;
+             long                datachannelId;
```

Comment:
fooId should refer to a stats object, for example trackId or trackIds, but identifiers that are not for stats objects but say the underlying webrtc object should have fooIdentifier, for example the MediaStream.id is surfaces as RTCMediaStreamStats.streamIdentifier to not be confused with the id of the RTCMediaStreamStats dictionary.

This should probably be changed to datachannelIdentifier and we should update the example in Guidelines for designing stats objects which use datachannelid as an example. This section should talk about "Identifiers"

Reply:
Or dataChannelIdentifier based on RTCDataChannel having capitalized C

Reply:
+1 dataChannelIdentifier

Reply:
Ah I see. I have changed it to `dataChannelIdentifier` then. I have also updated the "Names ending in "Id"" example to refer to `transportId` instead of `datachannelId` for consistency.